cloverage/project.clj

```diff
@@ -13,11 +13,11 @@
             :comments "same as Clojure"}
   :deploy-repositories [["releases" :clojars]
                         ["snapshots" :clojars]]
-  :dependencies [[org.clojure/tools.reader "1.3.2"]
+  :dependencies [[org.clojure/tools.reader "1.3.3"]
                  [org.clojure/tools.cli "0.4.2"]
                  [org.clojure/tools.logging "0.5.0"]
-                 [org.clojure/tools.namespace "0.3.1"]
-                 [org.clojure/java.classpath "0.3.0"]
+                 [org.clojure/tools.namespace "1.0.0"]
+                 [org.clojure/java.classpath "1.0.0"]
```

Comment:
A lot of other deps can/should be bumped as well, but I didn't do that as part of this PR